featuretests/cmd_juju_controller_test.go

```diff
@@ -106,7 +106,7 @@ func (s *cmdControllerSuite) TestAddModelNormalUser(c *gc.C) {
 		"\n"+
 		"Model           Cloud/Region        Status     Access  Last connection\n"+
 		"controller*     dummy/dummy-region  available  admin   just now\n"+
-		"test/new-model  dummy/dummy-region  available  admin   never connected\n"+
+		"test/new-model  dummy/dummy-region  available  -       never connected\n"+
```

Comment:
FWIW, this is a change back to whatever is currently in develop. One of the things I've done, forced my hand, and on your branch, I have originally modified the test to suit my needs. This is me changing it back...
However, I do agree that we should probably always return user access to the model. Even for superusers... I also wonder if we want to be explicit and tell superuser when they are 'superuser' on the model, i.e. no direct access was granted to this model but they can see it because they are 'gawds' on this controller.